index.html

```diff
@@ -27,47 +27,43 @@ <h2 class="card-title mt-0 h3">Delivering Digital Services</h2>
 
           <p class="card-text">A guide for using user-centred design and technology to deliver digital services.</p>
 
-          <a href="/digital" class="btn btn-success">Get started</a>
+          <a href="/digital" class="btn btn-success btn-sm">Get started</a>
         </div>
       </div>
     </div>
 
     <div class="col-md-6">
       <div class="card card-border rounded">
         <div class="card-body">
-          <h2 class="card-title mt-0 h3">Technology Guides</h2>
+          <h2 class="card-title mt-0 h3">Product</h2>
 
-          <p class="card-text">We have built a repository of learning materials, code katas, screencasts and guides for your technology team.</p>
+          <p class="card-text">We have outlined leading practices and resources for your product team.</p>
```

Comment:
Removing 'we have' from the sentences?

Reply:
Yeah, idk about that. I feel like the homepage should outline that this is what we think of as leading practices as Made Tech and as we get to a particular concept, it should be more informational. These are just my thoughts tho.

Would like to get more opinions on this.

Reply:
I don't mind about the "We have".

I think we should say that the product resources aren't just for "product teams". They are for anyone wanting to know about product management.

Reply:
I prefer the approach taken for the design section

Reply:
I prefer the approach taken for the design section.

Reply:
Updated in https://github.com/madetech/learn/pull/169/commits/96b305468ceb6e68f15070790e10a996c49a47f2.